deploy/deploy.js

```diff
@@ -1,14 +1,26 @@
 const exec = require('@actions/exec');
 
-async function deploy(environment, service, id) {
-  const args = [
-    'deploy',
-    environment,
-    service,
-    '--id', `${id}`,
-    '--quiet',
-    '--wait',
-  ];
+async function deploy(environment, service, id, idKey) {
+  let args;
+  if (idKey === 'ID') {
+    args = [
+      'deploy',
+      environment,
+      service,
+      '--id', `${id}`,
```

Comment:
you should change this to: `--${idKey}` and then remove the if

Reply:
or maybe you should do idKet.toLowerCase()? just to make sure its fit for a CLI command?